src/main/java/org/resitrack/util/CommonUtil.java

```diff
@@ -4,6 +4,6 @@ public class CommonUtil {
 
     public static final int INITIAL_ID_VALUE = 1;
     public static boolean isNullOrBlank(String input) {
-        return  input == null || input.isEmpty();
+        return  input == null || input.isEmpty() || input.trim().length() <= 1;
```

Comment:
we can trim the string before check whether it's empty or not, so we don't need to check the string length again.

```suggestion
        return  input == null || input.trim().isEmpty();
```